pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1377,6 +1377,43 @@ def func(ser):
     tm.assert_frame_equal(res, expected)
 
 
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        "int64",
+        "uint64",
+        "int32",
+        "uint32",
+        "float64",
+        "float32",
+        "Int64",
+        "UInt64",
+        "Int32",
+        "UInt32",
+        "Float64",
+        "Float32",
+    ],
+)
+def test_groupby_agg_precision(dtype):
```

Comment:
Could you use the `any_real_numeric_dtype` fixture here?

Reply:
Done. Just only used `any_real_numeric_dtype `.